addons/flags/CfgVehicles.hpp

```diff
@@ -0,0 +1,21 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Equipment {
+                class ADDON {
+                    displayName = CSTRING(ActionCategory);
+                    insertChildren = QUOTE(_this call FUNC(getActions));
```

Comment:
`_this` is implicitly passed
```suggestion
                    insertChildren = QUOTE(call FUNC(getActions));
```

Reply:
Personally I prefer the explicit `_this`, so it's up to PR author if they want to keep it or not.